modules/ROOT/pages/outbound-message-routing.adoc

```diff
@@ -1,6 +1,6 @@
 = Outbound Message Routing
 
-Partner Manager’s outbound message processing framework provides the flexibility to implement receive endpoints in a variety of ways. 
+Partner Manager’s outbound message processing framework provides the flexibility to implement receive endpoints in a variety of ways.
```

Comment:
Avoid making names possessive:
```suggestion
The Partner Manager outbound message processing framework enables you to implement receive endpoints in a variety of ways:
```
Reword list to work with this sentence as its intro., instead of using "You can:"